vulnerabilities.yaml

```diff
@@ -21,6 +21,25 @@
 #
 # Other tag: "ignore-python3: true", used for modules removed from Python 3
 
+- name: "bpo-35755: Remove current directory from posixpath.defpath"
+  bpo: 35755
+  disclosure: "2019-01-17 (Reported by bpo-35755)"
```

Comment:
disclosure, reported-at, reported-by and affected-versions are generated automatically.